frontend/src/pages/NewPlacePage.vue

```diff
@@ -377,14 +382,49 @@ export default {
             ]
         };
     },
+
+    created() {
+        this.$store.dispatch('placeCategory/getAllCategories')
+            .then((result) => {
+                this.categories = result;
+            });
+    },
+    
+    watch: {
+        'newPlace.category': function (categoryObject) {
+            if (categoryObject) {
+                this.newPlace.category_tags = [];
+                // this.$store.dispatch('placeCategoryTags/getTagsByCategoryId', categoryObject.id)
+                //     .then((result) => {
+                //         this.category_tags = result;
+                //     });
+                this.selectedTag = 'v';
+            }
+        },
+
+        'selectedTag': function (tagObject) {
+            if (tagObject !== 'v' && !this.newPlace.category_tags.some((tag) => tag.name === tagObject.name)) {
+                this.newPlace.category_tags.push(tagObject);
+                console.log(this.selectedTag.name);
+            }
+        }  
+    },
+
     computed: {
-        isCategoryNOTSelected: function () {
-            return !this.category;
+        isCategorySelected: function () {
+            return !!this.newPlace.category;
         },
 
         isDaySelected: function () {
             return !this.weekdays.length;
         }
+    },
+
+    methods: {
+        onCloseTab: function (tagObject) {
+            this.newPlace.category_tags = this.newPlace.category_tags.filter((tag) => tag.name !== tagObject.name);
```

Comment:
`this.newPlace.category_tags.filter((tag) => tag.name !== tagObject.name);` should be separate method